### DISCUSSION TITLE
Is there a model for BPMN standard?

### DISCUSSION THREAD

**Discussion by lapse00** [gaphor/gaphor#2296] [Q&A] [answered]
Hello,

Does anyone know if it is possible to use BPMN in gaphor? It would be great!

Thanks in advance.

**Comment by danyeaw** [comment, marked as answer]
Hi @lapse00, we don't currently support BPMN as a profile. It would be pretty straightforward if someone was interested in helping to contribute BPMN, the majority of it is UML activity diagrams.

**Reply by shobeira** [reply]
Hi Dany (@danyeaw)

I'm happy to help.

**Reply by danyeaw** [reply]
Hi @shobeira that would be awesome! A simple way to start would be to create a new BPMN profile based on UML and only show the General and Activity toolbars. Then overtime we could create a unique data model for it with the BPMN specific nomenclature
